### PR TITLE
Add dotnet-example to CI and a new `command` runner

### LINKED ISSUE #433: Run `dotnet-example`
https://github.com/FerretDB/dotnet-example

It is critical for us to stay compatible with C# driver. Let's integrate that test into our dance tool and CI.

### PR DESCRIPTION
**Pull request by b1ron**
Closes #433.

### CODE REVIEW

**Comment by b1ron** [comment]
I need to look into [this](https://github.com/FerretDB/dance/actions/runs/5058548343/jobs/9078810030?pr=442) failure.

**Review by AlekSi — DISMISSED** [review]
Let's integrate it into the dance tool as a new simple command runner

File: internal/command/command.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+func Run(ctx context.Context, dir string, args []string) (*internal.TestResults, error) {
+	// TODO https://github.com/FerretDB/dance/issues/20
+	_ = ctx
+
+	res := &internal.TestResults{
+		TestResults: make(map[string]internal.TestResult),
+	}
+
+	_ = dir
+	dir = args[len(args)-1]
```
Comment (AlekSi) [inline internal/command/command.go:38]: Why we ignored passed `dir` and use the last argument instead?

File: internal/command/command.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+	cmdName := args[0]
+	cmdArgs := args[1 : len(args)-1]
```
Comment (AlekSi) [inline internal/command/command.go:41]:
```suggestion
	cmdName, cmdArgs := args[0], args[1:]
```

File: internal/command/command.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+	for _, arg := range cmdArgs {
```
[the remark this answers is not in the document]
Comment (b1ron) [reply internal/command/command.go:43]: You are right, this was my strange attempt installing dependencies but the .csproj file will handle this when `dotnet run` is ran I believe.

File: internal/command/command.go (hunk 1 above)
Comment (b1ron) [reply internal/command/command.go:38]: To fix the linter, it complaines that the argument dir is overwritten before first use.

Comment (AlekSi) [reply internal/command/command.go:38]: Well, yeah, the linter is correct. Why we don't use passed dir?

**Review by w84thesun — DISMISSED** [review]
LGTM!

**Comment by b1ron** [comment]
@AlekSi PTAL when you get a chance.

**Review by AlekSi — APPROVED** [review]

**Pull request merged**